releasing/plugin-commands-deploy/src/createDeployFiles.ts

```diff
@@ -256,45 +257,47 @@ function convertResolvedDependencies (
 interface ResolveLinkOrFileResult {
   scheme: 'link:' | 'file:'
   resolvedPath: string
-  suffix?: `(${string})`
+  suffix?: string
 }
 
 function resolveLinkOrFile (spec: string, opts: Pick<ConvertOptions, 'lockfileDir' | 'projectRootDirRealPath'>): ResolveLinkOrFileResult | undefined {
```

Comment:
I see a couple of issues with this function. The argument here is called spec but it is executed with both depPath and specifiers even though they have different formats. DepPath will be `<pkgName>@<id><patchHash><peersSuffix>`. Specifier will be just a specifier that is present in package.json.

You probably only need to use depPath here. Which you can get from the importer.

Reply:
> You probably only need to use depPath here. Which you can get from the importer.

The `depPath` still needs to be transformed. When it encounter a workspace package `foo@packages/foo(suffix)`, it needs to be turned into `foo@file:///<absolute_path>/packages/foo(suffix)`

Reply:
I am not saying that transform is not required. But the source of the transform currently is incorrect. It shouldn't be the specifier.